File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/main/java/gameCatan/models/Computer.java
package gameCatan.models;

public class Computer extends Player {
    public Computer(int storageWood, int storageCoal, int storageSeed) {
        super(storageWood, storageCoal, storageSeed);
    }

    @Override
    public String toString() {
        return "Computer{" +
                "storageWood=" + storageWood +
                ", storageCoal=" + storageCoal +
                ", storageSeed=" + storageSeed +
                '}';
    }
}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/gameCatan/models/Computer.kt
package gameCatan.models

class Computer(storageWood: Int, storageCoal: Int, storageSeed: Int) : Player(storageWood, storageCoal, storageSeed) {
    override fun toString(): String {
        return "Computer{" +
                "storageWood=" + storageWood +
                ", storageCoal=" + storageCoal +
                ", storageSeed=" + storageSeed +
                '}'
    }
}